### ISSUE TITLE
Checkout - Segmentar checkout

### ISSUE THREAD

**Issue by amedtler** [amedtler/oceandrop#122]
Item 26 da planilha

**Comment by arielrinnert** [comment]
esse fica pro retorno

**Comment by arielrinnert** [comment]
eria igual aquele que você mostrou pra mim lembra? do longline..

só tem q acrescentar uma etapa adicional pegando o nome e e-mail do cara antes de qualquer coisa.. pq esse nome e email, já tem q ser salvos no servidor no momento que ele avança a etapa, pq se ele sair fora ali já temos q identificar via RD pra contatar ele..

Então tem q dividir o checkout em três etapas, sendo q só uma fica a mostra por vez.. e só pode sair da primeira etapa depois q ele já colocou um nome e e-mail.. (os forms 2 e 3 ficam bloqueados.. ai tem q ver se será preciso bloquear os campos do form 1 quando ele passar dessa etapa.. entendeu?)

aquela parte mais próxima do rodapé, com selos etc, pode ficar sempre a mostra..

**Comment by amedtler** [comment]
Identificador: pre-cadastro-checkout

**Comment by roelsmich** [comment]
@arielrinnert @amedtler se ligam no jeito que eles segmentaram o checkout. Simples, clean e tudo na mesma pagina

http://www.eyefolds.com/commerce/show-cart

**Comment by arielrinnert** [comment]
da hora esse modelo ai @roelsmich! tem a base pra implementar um esquema desse @amedtler?

**Comment by amedtler** [comment]
Bem legal mesmo. Porém não é woocommerce, como comentei contigo, temos algumas restrições quanto a disposição, etc...

**Comment by arielrinnert** [comment]
certo, certo, entendi.. @roelsmich vc quer dar uma verificada direto com o @amedtler nas possibilidades dentro do woocommerce?
